hack/build-bytecode-images-multi.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Set default value for IMAGE_NOO_BC tag if not already set
+IMAGE_NOO_BC=${IMAGE_NOO_BC:-quay.io/bpfman-bytecode/netobserv}
```

Comment:
shouldn't it be in netobserv org instead?
Something like `quay.io/netobserv/bpf-bytecode` ?

Reply:
`quay.io/netobserv/netobserv-bytecode` is ok ?

Reply:
`agent-bytecode`?
or `netobserv-agent-bytecode` ?

(mentioning either ebpf or agent to clarify its situation in the various netobserv components)

Reply:
I ended out using `quay.io/netobserv/ebpf-bytecode`